EasyReflectometryApp/Logic/Proxies/Project.py

```diff
@@ -112,213 +92,38 @@ def currentProjectPath(self, new_path):
 
     @Slot()
     def createProject(self):
-        projectPath = self.currentProjectPath
-        mainCif = os.path.join(projectPath, 'project.cif')
-        samplesPath = os.path.join(projectPath, 'samples')
-        experimentsPath = os.path.join(projectPath, 'experiments')
-        calculationsPath = os.path.join(projectPath, 'calculations')
-        if not os.path.exists(projectPath):
-            os.makedirs(projectPath)
-            os.makedirs(samplesPath)
-            os.makedirs(experimentsPath)
-            os.makedirs(calculationsPath)
-            with open(mainCif, 'w') as file:
-                file.write(self.projectInfoAsCif)
-        else:
-            print(f"ERROR: Directory {projectPath} already exists")
+        self._project.create_project_dir()
+        self._project.default_model()
+        self._relayProjectChange()
+        self.saveProject()
 
-    @Slot()
-    def saveProject(self):
-        self._saveProject()
+    def _relayProjectChange(self):
+        self.parent._model_proxy._model = self._project._models
```

Comment:
This makes `ProjectProxy` aware of other proxies. This should be avoided.
How about telling the main proxy to deal with the assignment?
E.g.
```python
self.parent.assign_model(self._project._models)
```
and the main proxy would just forward this to whichever proxy needs it, say
```python
def assign_model(self, model):
    self._model_proxy._model = model
```

Reply:
I expect that cases like this should disappear if we have `project` as the main data structure that all proxies know.  What I mean is that all the different proxies will be able to manipulate the `project` but each of the proxies  should be focused on a given part of `project`.

I will try to use that code structure in the `src_qt6` and leave the code as is for the "old" qt5 GUI.